src/couch/src/couch_file.erl

```diff
@@ -264,6 +265,18 @@ rename_file(Original) ->
         Else -> Else
     end.
 
+rename_dir(RootDelDir, Original, DbName) ->
+    DbDir = binary_to_list(DbName) ++ "_design",
+    DeletedIndexDir = filename:join(
```

Comment:
Minor nit, this should be RenamedIndexDir or future us might get confused why its called Deleted but isn't actually getting deleted.